3.5/deployment-cloud-aws.md

```diff
@@ -20,6 +21,12 @@ or using [Kubernetes](deployment-kubernetes.html),
 are all valid options to deploy on AWS. Please refer to the corresponding chapters for further 
 information.
 
-**Important:** In order to deploy on AWS, general guidelines, like using a fast,
-**direct-attached**, SSD disk for the data directory of the ArangoDB processes
-apply.
+{% hint 'info' %}
+In order to deploy on AWS, general guidelines like using a fast, **direct-attached**
+SSD disk for the data directory of the ArangoDB processes apply.
+
+We recommend [EC2 I3 instances](https://aws.amazon.com/ec2/instance-types/i3/){:target="_blank"}
+and to use **io1** storage rather than **gp2** on AWS because of the better throughput
```

Comment:
Suggest Instead:

... AWS, which are specifically designed with higher IOPS demands of databases in mind. Please keep in mind that replication by itself generates a manifold of the incoming write load in the back end.